rules/test.bzl

```diff
@@ -267,4 +298,18 @@ def ios_unit_snapshot_test(name, apple_library = apple_library, test_factory = d
         test_factory: Use this to generate other variations of tests.
         **kwargs: Arguments passed to the apple_library and ios_unit_test rules as appropriate.
     """
-    _ios_test(name, _ios_internal_unit_test_bundle, _ios_unit_test, test_factory, apple_library, internal_test_deps = ["@bazel_tools//tools/cpp/runfiles"], **kwargs)
```

Comment:
This is the bulk of the change, `@bazel_tools//tools/cpp/runfiles` was set for all snapshot test targets for some reason... this should be set by snapshot targets which need it not on in a global manner